packages/owl-bot/src/bin/commands/commit-post-processor-update.ts

```diff
@@ -80,10 +80,28 @@ export const commitPostProcessorUpdateCommand: yargs.CommandModule<{}, Args> = {
         default: cwd(),
       });
   },
-  handler: argv => commitPostProcessorUpdate(argv),
+  handler: async argv => {
+    const {shouldPromoteFromDraft} = await commitPostProcessorUpdate(argv);
+    if (shouldPromoteFromDraft) {
+      await promoteFromDraft(shouldPromoteFromDraft, argv['github-token']);
+    }
+  },
 };
 
-export async function commitPostProcessorUpdate(args: Args): Promise<void> {
+interface PRLocator {
+  owner: string;
+  repo: string;
+  pull_number: number;
+}
+
+interface AfterCommitPostProcessorUpdate {
+  /// When present, the pull request should be marked ready for review.
+  shouldPromoteFromDraft?: PRLocator;
```

Comment:
nit: this field name implies a boolean return type. Perhaps a name like `pullRequestToPromote` or something like that?

Reply:
done